From c61daf24bfddcb2e5c35b6967697423d60934323 Mon Sep 17 00:00:00 2001
From: Johan Hovold <jhovold@gmail.com>
Date: Mon, 29 Oct 2012 10:56:25 +0100
Subject: [PATCH 08/12] USB: serial: export usb_serial_generic_chars_in_buffer
References: bnc#956375
Git-Commit: 428d9988557f0f26047af304cf1f3d130b06ed4d
Patch-Mainline: v3.8

Export generic chars_in_buffer implementation so it can be used in
subdrivers in combination with checks of any hardware buffers.

Signed-off-by: Johan Hovold <jhovold@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/usb/serial/generic.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/serial/generic.c b/drivers/usb/serial/generic.c
index 9f0b2bf..ad03a2c 100644
--- a/drivers/usb/serial/generic.c
+++ b/drivers/usb/serial/generic.c
@@ -309,6 +309,7 @@ int usb_serial_generic_chars_in_buffer(struct tty_struct *tty)
 	dbg("%s - returns %d", __func__, chars);
 	return chars;
 }
+EXPORT_SYMBOL_GPL(usb_serial_generic_chars_in_buffer);
 
 int usb_serial_generic_submit_read_urb(struct usb_serial_port *port,
 					gfp_t mem_flags)
-- 
2.1.4

